app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> float:
```

Comment:
Indicate a list of what instances you expect to obtain for `players` in the type annotation here and for `elves` and `dwarfs` in the `elves_concert` and `feast_of_the_dwarves` functions correspondingly